### PR TITLE
media-gfx/inkscape: version bump to 1.3

### PR DESCRIPTION
**Pull request by t0b3**

### CODE REVIEW

File: media-gfx/inkscape/inkscape-1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	test? ( virtual/imagemagick-tools )
+"
+COMMON_DEPEND="${PYTHON_DEPS}
+	>=app-text/poppler-0.57.0:=[cairo]
+	>=dev-cpp/cairomm-1.12:0
+	>=dev-cpp/glibmm-2.54.1:2
+	dev-cpp/gtkmm:3.0
+	>=dev-cpp/pangomm-2.40:1.4
+	>=dev-libs/boehm-gc-7.1:=
+	dev-libs/boost:=
```
Comment (thesamesam) [inline media-gfx/inkscape/inkscape-1.3.ebuild:42]: Needs a [stacktrace(-)] dep for boost (just rebase on 9999 as I pushed the change there a moment ago).

Comment (t0b3) [reply media-gfx/inkscape/inkscape-1.3.ebuild:42]: @thesamesam really? It compiles and runs totally fine here even with boost with USE=stacktrace...

Comment (t0b3) [reply media-gfx/inkscape/inkscape-1.3.ebuild:42]: @thesamesam sorry I realize I just got it the other way round... :+1: so I will add this dep, sure

Comment (t0b3) [reply media-gfx/inkscape/inkscape-1.3.ebuild]: @thesamesam done

**Pull request closed without merging**